casbin/enforcer.py

```diff
@@ -85,3 +87,19 @@ def get_permissions_for_user(self, user):
     def has_permission_for_user(self, user, *permission):
         """determines whether a user has a permission."""
         return self.has_policy(*permission)
+
+    def get_implicit_roles_for_user(self, user):
+        roles = self.get_roles_for_user(user)
```

Comment:
`get_implicit_roles_for_user` should have the second parameter `domain`.
Then, please add a comment.

Reply:
I added a function get_implicit_roles_for_user_in_domain, and comments.